DBM-Raids-Vanilla/VanillaSoD_SunkenTemple/JammalanAndOgom.lua

```diff
@@ -149,15 +150,15 @@ function mod:SPELL_CAST_SUCCESS(args)
 end
 
 function mod:SPELL_AURA_APPLIED(args)
-	local spellId = args.spellId
-	if spellId == 437809 then
+	if args.spellId == 437809 then
 		warnHolyFire:CombinedShow(0.3, args.destName)--On multiple targets?
-	elseif spellId == 437847 then
+	elseif args.spellId == 437847 then
 		warnMortalLash:Show(args.destName)
-	elseif spellId == 437927 then
+	elseif args.spellId == 437927 then
 		warnShadowSermonPain:CombinedShow(0.3, args.destName)--On multiple targets?
-	elseif spellId == 437930 then
+	elseif args.spellId == 437930 then
 		warnPowerWordShield:Show(args.destName)
+		specWarnPowerWordShieldPurge:Show(args.destName)
```

Comment:
Added special warning object and set voice version of 2, but didn't actually add a voice call via Play (which would probably use "dispelboss" here)

Reply:
I also like to avoid double chat frame spam, so in this case I might do something like
if self.Options.SpecWarn12345dispel then
show dispel warning
else
show target warning
end